test/integration_tests/wav2vec2_pipeline_test.py

```diff
@@ -61,7 +61,6 @@ def test_pretraining_models(bundle):
 )
 def test_finetune_asr_model(
         bundle,
-        lang,
```

Comment:
This is not requirement for the CI job, but it's not really necessary either so I removed it.